src/libraries/StateLibrary.sol

```diff
@@ -267,7 +267,7 @@ library StateLibrary {
 
     /**
      * @notice Retrieves the liquidity of a position.
-     * @dev Corresponds to pools[poolId].positions[positionId].liquidity. A more gas efficient version of getPositionInfo
+     * @dev Corresponds to pools[poolId].positions[positionId].liquidity. More gas efficient for just retrieiving liquidity as compared to getPositionInfo
```

Comment:
Just missing the `calulcate` change on line 216

Reply:
ah oops good catch!